src/pretix/plugins/stripe/payment.py

```diff
@@ -698,59 +778,33 @@ def shred_payment_info(self, obj: OrderPayment):
                 le.save(update_fields=['data', 'shredded'])
 
 
-class StripeCC(StripeMethod):
-    identifier = 'stripe'
-    verbose_name = _('Credit card via Stripe')
-    public_name = _('Credit card')
-    method = 'cc'
-
-    def payment_form_render(self, request, total) -> str:
-        account = get_stripe_account_key(self)
-        if not RegisteredApplePayDomain.objects.filter(account=account, domain=request.host).exists():
-            stripe_verify_domain.apply_async(args=(self.event.pk, request.host))
-
-        template = get_template('pretixplugins/stripe/checkout_payment_form_cc.html')
-        ctx = {
-            'request': request,
-            'event': self.event,
-            'total': self._decimal_to_int(total),
-            'settings': self.settings,
-            'is_moto': self.is_moto(request)
-        }
-        return template.render(ctx)
+class StripePaymentIntentMethod(StripeMethod):
+    identifier = ''
+    method = ''
 
     def payment_is_valid_session(self, request):
-        return request.session.get('payment_stripe_payment_method_id', '') != ''
```

Comment:
Changing this and similar things is of course possible and creates a cleaner codebase, but we have a risk to think about with people who are performing a checkout while we ware deploying the change. They might have already entered their card number and then the session gets lost during purchase? We might want to deploy this at night, but still, people are buying with Stripe all the time. (Similar concerns with the PayPal PR.)